test/index.test.js

```diff
@@ -6,4 +6,8 @@ test('adds 1 + 2 to equal 3', () => {
 
 test('adds 5 + 9 to equal 14', () => {
     expect(add(5, 9)).toBe(14);
+});
+
+test('adds 5 + 10 to equal 15', () => {
+    expect(add(5, 10)).toBe(15);
 });
```

Comment:
This code is Simmiler in privuas test case

Reply:
But  What i need to do...

Reply:
You need to put some thing to that is failed

Reply:
ok i will try my best